eofconvention.fsx

```diff
@@ -0,0 +1,26 @@
+open System.IO
+
+let EolAtEof(path: FileInfo) = 
+
+    use streamReader = new StreamReader (path.FullName)
+    let filetext = streamReader.ReadToEnd()
+    
+    if filetext <> "" then
+        (filetext |> Seq.last = '\n')
+    else
+        true
+
+let NotPrefix (prefix: string) (mainStr: string) =
```

Comment:
mainStr still a confusing name, maybe use the same as you have used in line 20?

Reply:
Done